sdk/core/src/services/github/client/mod.rs

```diff
@@ -104,10 +115,31 @@ impl Client {
             }
         };
 
-        match gh_commits_rsp.last_hash {
-            Some(val) => Ok(Some(val)),
-            None => Ok(None),
-        }
+        Ok(gh_commits_rsp.last_hash)
+    }
+
+    /// Gets the latest release tag name from the GitHub repo
+    pub async fn get_latest_release(&self, repo: &Repo) -> Result<String, Error> {
```

Comment:
```suggestion
    pub async fn get_latest_release_tag(&self, repo: &Repo) -> Result<String, Error> {
```

Just to make it explicit like the docstring is.  What do you think?